source/install/verify-signatures.txt

```diff
@@ -0,0 +1,176 @@
+.. _verify-signatures-mongosh:
+
+==========================================
+Verify Integrity of MongoDB Shell Packages
+==========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. include:: /includes/verify-signatures-intro.rst
+
+Before you Begin
+----------------
+
+If you don't have MongoDB Shell installed, download the MongoDB Shell
+binary from the `Download Center
+<https://www.mongodb.com/try/download/shell?jmp=docs>`__.
+
+Steps
+-----
+
+To verify MongoDB Shell packages, you can use one of these methods:
+
+- :ref:`Disk Image Verification <verify-signatures-disk-image>`
+
+- :ref:`GPG Verification <verify-signatures-gpg>`
+
+- :ref:`RPM Package Verification <verify-signatures-rpm>`
+
+.. _verify-signatures-disk-image:
+
+Verify Packages with Disk Image Verification
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run this command:
+
+.. code-block:: sh
+
+   codesign -dv --verbose=4 <path_to_mongosh_executable>
+
+If the package is properly signed by MongoDB, the output includes the
+following information:
+
+.. code-block:: sh
+   :copyable: false
+
+   Authority=Developer ID Application: MongoDB, Inc. (4XWMY46275)
+   Authority=Developer ID Certification Authority
+   Authority=Apple Root CA
+
+.. _verify-signatures-gpg:
+
+Verify Packages with GPG
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. procedure::
+   :style: normal
+
+   .. step:: Import the MongoDB Shell public key
+
+      Run this command:
+
+      .. code-block:: sh
+
+         curl https://pgp.mongodb.com/mongosh.asc | gpg --import
+
+      If the key imports successfully, the command returns the
+      following output:
+
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: key CEED0419D361CB16: public key "Mongosh Release Signing Key <packaging@mongodb.com>" imported
+         gpg: Total number processed: 1
+         gpg:               imported: 1
+
+      If you have previously imported the key, the command returns the
+      following output:
+
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: key A8130EC3F9F5F923: "Mongosh Release Signing Key <packaging@mongodb.com>" not changed
+         gpg: Total number processed: 1
+         gpg:             	      unchanged: 1
```

Comment:
We can remove the extra space in the second line to that `:1` is aligned in both.